src/main/java/com/jcabi/log/Logger.java

```diff
@@ -368,6 +388,16 @@ public void write(final int data) throws IOException {
         } else {
             logger = LoggerFactory.getLogger(source.getClass());
         }
+        try {
+            final Field fqcn = logger.getClass()
+                .getDeclaredField("FQCN");
+            setFinalStatic(fqcn, Logger.class.getName());
+            // @checkstyle EmptyBlockCheck (5 lines)
+        } catch (final NoSuchFieldException ex) {
+        } catch (final SecurityException ex) {
+        } catch (final IllegalArgumentException ex) {
```

Comment:
I don't want that it throws any exception if it fails. It will fail if a client use non-log4j binder. This issue is related only to log4j binding.